samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -239,5 +239,58 @@ public class Assignment
                 throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
             }
         }
+
+        /// <summary>
+        /// Update an assignment
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <returns>EducationAssignment</returns>
+        public static async Task<EducationAssignment> PatchAsync(
+            GraphServiceClient client,
+            string classId,
+            string assignmentId)
+        {
+            try
+            {              
+                var requestBody = new EducationAssignment
+                {
+                    DisplayName = "Reading and review test updated",                    
+                };
+
+                return await client.Education
+                    .Classes[classId]
+                    .Assignments[assignmentId]
+                    .PatchAsync(requestBody);
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"PatchAsync call: {ex.Message}", ex, classId, assignmentId);
+            }
+        }
+
+        /// <summary>
+        /// Delete an assignment
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <returns></returns>
+       
+        public static async Task DeleteAsync(
+            GraphServiceClient client,
+            string classId,
+            string assignmentId)
+        {
+            try
+            {               
+                await client.Education.Classes[classId].Assignments[assignmentId].DeleteAsync();          
+            }
+            catch (Exception ex)
```

Comment:
what happens when there is no assignment with given id existing in the given class? shall we log 404 and not throw exception.

Reply:
We are trying to delete the assignment created above due to that we might not see the above scenario. 
Probably we can cover error handling in a separate PR.